clusterctl/examples/skeleton/machines.yaml.template

```diff
@@ -6,6 +6,9 @@ items:
     labels:
       set: controlplane
   spec:
+    versions:
+      kubelet: 1.10.6
+      controlPlane: 1.10.6
 - apiVersion: "cluster.k8s.io/v1alpha1"
   kind: Machine
   metadata:
```

Comment:
Spec for node machine also requires the versions field,

Reply:
Good point.